src/test/librados/io_cxx.cc

```diff
@@ -846,7 +846,7 @@ TEST_F(LibRadosIoECPP, RmXattrPP) {
 
 TEST_F(LibRadosIoECPP, CrcZeroWrite) {
   SKIP_IF_CRIMSON();
-  set_allow_ec_overwrites(pool_name, true);
+  set_allow_ec_overwrites();
```

Comment:
The new commit lacks the `cherry-picked from ` information (`-x` for `git cherry-pick`).

Reply:
fixed that